client/src/components/layout/UserShow.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import UserSerializer from "../../../../server/src/serializers/UserSerializer";
+import BuildsList from "./BuildsList";
+import ReviewsList from "./ReviewsList"
+
+const UserShow = ({user}) => {
+    
+    const [currentUserData, setUserData] = useState({
+        buildsList: [],
+        reviewsList: []
+    })
+
+    const userInfo = UserSerializer.serializeUser(user)
```

Comment:
I have not seen this in a while haha.

I have no idea how this is even working, but you should keep every file in the backend to the backend alone. that means no user serializer here.